echo.go

```diff
@@ -496,6 +496,7 @@ func (common) static(prefix, root string, get func(string, HandlerFunc, ...Middl
 		}
 		return c.File(name)
 	}
+	get(prefix, h)
```

Comment:
LGTM
Maybe you can add a new unit test, to avoid future regressions

```go
func TestEchoStaticRedirectIndex(t *testing.T) {
	assert := assert.New(t)
	e := New()

	// HandlerFunc
	e.Static("/static", "_fixture")

	errCh := make(chan error)

	go func() {
		errCh <- e.Start("127.0.0.1:1323")
	}()

	time.Sleep(200 * time.Millisecond)

	if resp, err := http.Get("http://127.0.0.1:1323/static"); err == nil {
		defer resp.Body.Close()
		assert.Equal(http.StatusOK, resp.StatusCode)

		if body, err := ioutil.ReadAll(resp.Body); err == nil {
			assert.Equal(true, strings.HasPrefix(string(body), "<!doctype html>"))
		} else {
			assert.Fail(err.Error())
		}

	} else {
		assert.Fail(err.Error())
	}

	if err := e.Close(); err != nil {
		t.Fatal(err)
	}
}
```

Reply:
yes,  I'll fix the Directory Redirect test case and add a TestEchoStaticRedirectIndex func for strengthening.